frontend/micro-ui/web/micro-ui-internals/packages/components-core/src/atoms/ButtonSelector.js

```diff
@@ -20,7 +20,9 @@ const ButtonSelector = (props) => {
       disabled={props.isDisabled}
       style={props.style ? props.style : null}
     >
-      <h2 style={{ ...props?.textStyles, ...{ width: "100%" } }}>{props.label}</h2>
+      <h2 style={{ ...props?.textStyles }} className="digit-button-label">
```

Comment:
@nabeelmd-eGov  just see what is difference between button , button selector and submitbar 
you can have a wrapper component but should not use <button> again

Reply:
@jagankumar-egov replaced with button component